src/Pages/SearchResult/SearchResult.js

```diff
@@ -1,13 +1,82 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import "./SearchResult.scss";
 
-class componentName extends Component {
+class SearchResult extends Component {
   render() {
     return (
-      <div>
-        
+      <div className="SearchResult">
+        <div className="noticeContainer">
+          <span className="title">루프탑</span>
+          <span className="resultText">(으)로 검색한 결과입니다.</span>
+          <div className="selectBarContainer">
+            <div className="optionContainer">
+              <div className="optionBox">
+                <span className="optionType">공간 유형</span>
+                <div className="optionWrapper">
+                  <span className="option">모든 공간</span>
+                  <div className="downArrow"></div>
+                </div>
+              </div>
+              <div className="optionBox">
+                <span className="optionType">지역</span>
+                <div className="optionWrapper">
+                  <span className="option">전체</span>
+                  <div className="downArrow"></div>
+                </div>
+              </div>
+              <div className="optionBox">
+                <span className="optionType">이용일</span>
+                <div className="optionWrapper">
+                  <span className="option">모든 날짜</span>
+                  <div className="calendarIcon"></div>
+                </div>
+              </div>
```

Comment:
이 부분은 map을 활용하면 확 줄일 수 있을 거 같애요!
```suggestion
const OPTIONS = [{id: "1", type: "공간 유형", option: "모든 공간"},
{id: "2", type: "지역", option: "전체"},
{id: "3", type: "이용일", option: "모든 날짜"}]
{OPTIONS.map(el => {return(
              <div className="optionBox" key={el.id}>
                <span className="optionType">{el.type}</span>
                <div className="optionWrapper">
                  <span className="option">{el.option}</span>
                  <div className="downArrow" />
                </div>
              </div>)})}
```
그리고 아무것도 감싸고 있지 않은 태그는 self closing 으로 하시면 돼요!
map 연습용으로 한 번 참고해보세요!

Reply:
네! 수정하였습니다!